sdk/aqueduct/aqueduct_client.py

```diff
@@ -431,11 +441,31 @@ def delete_flow(self, flow_id: Union[str, uuid.UUID]) -> None:
             InternalServerError:
                 An unexpected error occurred within the Aqueduct cluster.
         """
+        if writes_to_delete is None:
+            writes_to_delete = defaultdict()
         flow_id = parse_user_supplied_id(flow_id)
 
         # TODO(ENG-410): This method gives no indication as to whether the flow
         #  was successfully deleted.
-        self._api_client.delete_workflow(flow_id)
+        resp = self._api_client.delete_workflow(flow_id, writes_to_delete, force)
+
+        failed_deletions = {}
+        counts = 0
+        for integration in resp.writes_results:
+            failed_for_integration = []
+            for obj in resp.writes_results[integration]:
+                if not obj.succeeded:
+                    failed_for_integration.append(obj)
+            if len(failed_for_integration) > 0:
+                failed_deletions[str(integration)] = [
+                    res.__dict__ for res in failed_for_integration
+                ]
+                counts += len(failed_for_integration)
+        if counts > 0:
+            print("Workflow-Written Objects' Deletion Failures")
+            print(f"{counts} Failures")
+            print(json.dumps(failed_deletions, sort_keys=False, indent=4))
```

Comment:
this will print the UUID as keys. I feel we would also want to print the integration name instead.